### ISSUE TITLE
save teacher logits to be used with knowledge distillation

### ISSUE THREAD

**Issue by Jose-Velasco** [Jose-Velasco/brain-tumor-ai-pipeline#3]

**Comment by Jose-Velasco** [comment]
some type of utility function we can use for our trained teacher and the pre-trained teacher
